### ISSUE TITLE
Don't turn down music during NPC dialogue

### ISSUE THREAD

**Issue by nmadadair** [TwilioQuest/owls-nest-extension#7]
1.) Now that most of the NPC dialogue does not have voiceover, we probably shouldn't turn down the music when a dialogue window pops up anymore. The music getting soft during conversations feels weird without spoken lines.
